exonum-java-binding-qa-service/src/main/java/com/exonum/binding/qaservice/QaServiceImpl.java

```diff
@@ -134,44 +134,48 @@ public void afterCommit(BlockCommittedEvent event) {
   @Override
   public HashCode submitCreateCounter(String counterName) {
     CreateCounterTx tx = new CreateCounterTx(counterName);
-    return submitTransaction(tx);
+    return submitTransaction(CreateCounterTx.converter().toRawTransaction(tx));
```

Comment:
I wonder if we shall drop Converters, and replace them with static factory methods of `RawTransaction`s, something like `IncrementCounterTx#createRawTransaction(long, HashCode)`, because creating a transaction only to convert it to a raw transaction seems a little strange.

However, as eventually the test driver will submit _most_ transaction _directly_ to core, we may leave it as is, but add a todo here.